extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -326,152 +361,155 @@ public function doWikiForm() {
 			"protected"   => WikiFactory::getFlags ( $this->mWiki->city_id ) & WikiFactory::FLAG_PROTECTED,
 			"statuses"    => $this->mStatuses,
 			"variables"   => WikiFactory::getVariables(),
-			"variableName"=> $this->mVariableName,
+			"variableName" => $this->mVariableName,
 			"isDevel"     => $wgDevelEnvironment,
-			'wikiFactoryUrl' => Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullUrl(),
+			'wikiFactoryUrl' => Title::makeTitle( NS_SPECIAL, 'WikiFactory' )->getFullURL(),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikiFactoryUrl" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)